lib/get_parser.py

```diff
@@ -479,6 +479,13 @@ def get_parser() -> argparse.ArgumentParser:
         " sequential $JOBSUBJOBSECTION. Scripts may need modification to take"
         " this into account",
     )
+    parser.add_argument(
+        "--managed-token",
+        action="store_const",
+        const=True,
+        default=False,
+        help="Optimize calls to condor_vault_storer, etc.",
```

Comment:
Maybe add `when token is managed` to the help.

Reply:
Yes - agreed.  I think the help ought to be clear that this is the flag to use when we assume that the vault token will be managed external to jobsub.